eo-runtime/src/test/eo/org/eolang/txt/text-tests.eo

```diff
@@ -70,3 +71,45 @@
           * " " " "
       * "start" "end"
     $.equal-to "start & end"
+
+[] > text-lower-case
```

Comment:
@EugeneDar would be much better to make individual test cases instead of grouping them like this.

Reply:
@yegor256 sure, will fix it.